README.md

```diff
@@ -47,13 +52,52 @@ notebook_login()
 ```
 huggingface-cli login
 ```
-Then:
+Then
+
+**With `package_to_hub()`**:
+
+```python
+import gym
+
+from stable_baselines3 import PPO
+from stable_baselines3.common.evaluation import evaluate_policy
+from stable_baselines3.common.vec_env import VecVideoRecorder, DummyVecEnv
+from huggingface_sb3_test import load_from_hub, package_to_hub
+
+# Create the environment
+env_id = 'LunarLander-v2'
+env = DummyVecEnv([lambda: gym.make(env_id)])
+
+# Create the evaluation env
+eval_env = DummyVecEnv([lambda: gym.make(env_id)])
+
+# Instantiate the agent
+model = PPO('MlpPolicy', env, verbose=1)
+
+# Train the agent
+model.learn(total_timesteps=int(5000))
+
+# This method save, evaluate, generate a model card and record a replay video of your agent before pushing the repo to the hub
+package_to_hub(model=model, 
+               model_name="ppo-LunarLander-v2",
+               model_architecture="PPO",
+               env_id=env_id,
+               eval_env = eval_env,
+               repo_id="ThomasSimonini/TEST2-Colab-ppo-LunarLander-v2",
+               commit_message="Test commit")
+```
+
+
+**With `push_to_hub()`**:
```

Comment:
Maybe here add some explanation that this will only upload the `zip` file.

Reply:
:heavy_check_mark: